src/pydap/tests/test_cas_esgf.py

```diff
@@ -0,0 +1,71 @@
+from pydap.client import open_url
+import pydap.net
+from pydap.cas import esgf
+import requests
+import numpy as np
+import os
+import logging
+
+import sys
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+logging.basicConfig(filename='cas_esgf_test.log', level=logging.DEBUG)
+
+
+class TestESGF(unittest.TestCase):
+    url = ('http://cordexesg.dmi.dk/thredds/dodsC/cordex_general/'
+           'cordex/output/EUR-11/DMI/ICHEC-EC-EARTH/historical/r3i1p1/'
+           'DMI-HIRHAM5/v1/day/pr/v20131119/'
+           'pr_EUR-11_ICHEC-EC-EARTH_historical_r3i1p1_'
+           'DMI-HIRHAM5_v1_day_19960101-20001231.nc')
+
+    def test_basic_esgf_auth(self):
+        """Set up PyDAP to use the URS request() function"""
+        session = esgf.setup_session(os.environ['OPENID_ESGF'],
```

Comment:
Definitely useful! I think though, that I'd rather not connect my personal auth credentials with a joint/organizational account (e.g. Pydap), so could you disable the auth tests in the Travis run? After you do that and push, we should have all green lights at the end of this PR.